regexes/device/mobiles.yml

```diff
@@ -81,9 +81,15 @@ HTC:
 
     - regex: 'NexusHD2' # custom rom for hd2
       model: 'HD2'
-    - regex: 'HTC[ _\-]((?:Flyer|P715a).*) Build'
+    - regex: 'HTC[ _\-]P715a'
       device: 'tablet'
-      model: '$1'
+      model: 'P715a'
+    - regex: 'HTC[ _\-]Flyer Build'
+      device: 'tablet'
+      model: 'Flyer'
+    - regex: 'HTC[ _\-]Flyer[ _\-]([\w]{1,5})'
+      device: 'tablet'
+      model: 'Flyer $1'
     - regex: 'HTC[ _\-]One[ _\-]max[);/ ]'
```

Comment:
Hm. thats a good question ;-) Guess the range of 5.5–6.99 inches is fine. Wasn't sure about counting 7'' as phablet or not. But we can leave it like it is.